open-api/rest-catalog-open-api.yaml

```diff
@@ -2800,6 +3062,58 @@ components:
           additionalProperties:
             type: string
 
+    PrePlanTableRequest:
+      type: object
+      required:
+        - select
+        - options
+      properties:
+        select:
+          description: A list of the selected columns that user specifies during a query.
+          type: array
+          items:
+            type: string
+        filter:
+          $ref: '#/components/schemas/Expression'
+        options:
+          type: object
+          additionalProperties:
+            type: string
+        snapshot-id:
+          type: integer
+          format: int64
+        timestamp-ms:
```

Comment:
missing ref

Reply:
Not sure what you mean by this? From the proposal seems to match https://docs.google.com/document/d/1FdjCnFZM1fNtgyb9-v9fU4FwOX4An-pqEwSaJe8RgUg/edit

Reply:
I am talking about this: https://github.com/apache/iceberg/blob/main/api/src/main/java/org/apache/iceberg/TableScan.java#L48

Reply:
will add this